src/components/rich-text-editor/adapter/vue-froala.ts

```diff
@@ -203,17 +203,16 @@ export enum FroalaStatus {
     }
 
     protected addImageButton(): void {
-        const EDITOR_INSTANCE: VueFroala = this;
-
         $.FroalaEditor.RegisterCommand('insertImage', {
             title: this.$i18n.translate('m-rich-text-editor:insert-image'),
             undo: true,
             focus: true,
             showOnMobile: true,
-            callback: (): void => {
-                EDITOR_INSTANCE.allowedExtensions = EDITOR_INSTANCE.imageExtensions;
-                EDITOR_INSTANCE.isFileUploadOpen = true;
-                EDITOR_INSTANCE.selectedImage = undefined;
+            callback: function(): void {
+                let currentInstance: VueFroala = this.$oel[0].parentNode.__vue__;
```

Comment:
J'aime pas trop cette ligne, Je suggère de sortir le selecteur d'instance dans une méthode et peut-être faire des check pour les null ou les array out of bound .

Reply:
C'est ce que je voulais faire au début, mais le this ici représente l'instance de l'éditeur Froala et non le component Vue. C'est pour ça que je dois faire un truc laid comme ça...